public/css/main.css

```diff
@@ -316,6 +316,9 @@ body.darkBG .progress {
 .trailing-zeroes {
   opacity: .3
 }
+.decimal.trailing-zeroes {
+  margin-left: 0 !important;
+}
```

Comment:
@gozart1 Is this an OK way to avoid getting the -2px left margin from `.decimal`?